src/pages/index.js

```diff
@@ -43,10 +89,28 @@ const profileEditModal = new PopupWithForm({
 profileEditModal.setEventListeners();
 
 const addNewCardButton = document.querySelector("#profile-add-button");
+
 const addCardModal = new PopupWithForm({
   popupSelector: "#add-card-modal",
-  handleFormSubmit: handleAddCardFormSubmit,
+  handleFormSubmit: (data) => {
+    addCardModal.setLoading(true, "Saving...");
+    api
+      .addCardModal(data)
+      .then((res) => {
+        cardSection.addItems(createCard(res));
```

Comment:
`        cardSection.addItems(createCard(res));
`

Please, reuse the `renderCard` function, which you have in line 69, for creating cards, to avoid code duplication